pkg/sendbuffer/sendbuffer.go

```diff
@@ -161,7 +162,8 @@ func (sb *SendBuffer) sendAndPurge() error {
 	defer sb.sendMutex.Unlock()
 
 	toSendBuff := &bytes.Buffer{}
-	if err := sb.flushToWriter(toSendBuff); err != nil {
+	lastKey, err := sb.copyLogs(toSendBuff, sb.maxSendSize)
```

Comment:
if `toSendBuff` is larger than `maxSendSize` will this cause logs to be split mid-line?

Reply:
no, copyLogs peeks at the next logs size before adding and returns if adding it would exceed max size, I updated the copy logs comment.